DER/FxDerivatives/FxOptions.rdf

```diff
@@ -64,193 +68,109 @@
 		<rdfs:label xml:lang="en">allows for option of</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-der-fx-opt;CashCallOption"/>
 		<rdfs:range rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementCommitment"/>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;buyer">
-		<rdfs:label xml:lang="en">buyer</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-opt;ForeignExchangeOption"/>
-		<rdfs:range rdf:resource="&fibo-der-fx-opt;ForeignExchangeOptionBuyer"/>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;buyerIsParty">
-		<rdfs:label xml:lang="en">buyer is party</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-opt;ForeignExchangeOptionTransaction"/>
-		<rdfs:range rdf:resource="&fibo-der-fx-opt;ForeignExchangeOptionBuyer"/>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;buyingCurrency">
-		<rdfs:label xml:lang="en">buying currency</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-opt;OTCCurrencyOption"/>
-		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;Currency"/>
-		<skos:definition xml:lang="en">The currency in which the option is bought.</skos:definition>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;establishes">
-		<rdfs:label xml:lang="en">establishes</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-opt;FxCallOptionContract"/>
-		<rdfs:range rdf:resource="&fibo-der-fx-opt;CashCallOption"/>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;establishes.1">
-		<rdfs:label xml:lang="en">establishes</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-opt;FxCallOptionContract"/>
-		<rdfs:range rdf:resource="&fibo-der-der-fwd;ForwardDeliveryCommitment"/>
-	</owl:ObjectProperty>
-	
 	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;expiry">
 		<rdfs:label xml:lang="en">expiry</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-der-fx-opt;ForeignExchangeOption"/>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;DateTime"/>
 		<skos:definition xml:lang="en">FpML &apos;The date and time in a location of the option expiry. In the case of american options this is the latest possible expiry date and time.&apos;</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;hasCorrespondingContract">
-		<rdfs:label xml:lang="en">has corresponding contract</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-opt;ForeignExchangeOptionTransaction"/>
-		<rdfs:range rdf:resource="&fibo-der-fx-opt;ForeignExchangeOption"/>
+	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;hasBuyingCurrency">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasCurrency"/>
+		<rdfs:label xml:lang="en">has buying currency</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-fx-opt;CurrencyOption"/>
+		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;Currency"/>
+		<skos:definition xml:lang="en">The currency in which the option is bought.</skos:definition>
```

Comment:
ambiguous: is it for buying the option or buying the cash when the option is exercised? I could buy an option in USD that gives me the future right to buy GBP in EUR